contracts/DankFaucet.sol

```diff
@@ -31,12 +31,11 @@ contract DankFaucet is usingOraclize, DSAuth {
   uint public allotment;
 
   // An event for letting the caller know that they didn't supply enough
-  // ETH to complete the transaction.
-  event NotEnoughETH(string description, uint ethRequired);
+  // ETH/DANK to complete the transaction.
+  event NotEnoughFunds(string description, uint balance, uint balanceRequired);
 
-  // An event for communicating success/failure of the steps of the
-  // verification process, including calls to the Oraclize service.
-  event DankEvent(bytes32 hashedPhoneNumber, bool successful, string description);
+  // An event for communicating successful transfer of funds
+  event DankEvent(string description, bytes32 hashedPhoneNumber);
 
   // An event for communicating the id from Oraclize and a message.
   event OraclizeCall(bytes32 id, string msg);
```

Comment:
what is the purpose of this event?

Reply:
Logging mainly - we can follow up with Oraclize/Provable if sth went wrong